server/src/internalClusterTest/java/org/elasticsearch/action/search/TransportSearchIT.java

```diff
@@ -131,9 +132,10 @@ public void testLocalClusterAlias() {
         indexRequest.setRefreshPolicy(WriteRequest.RefreshPolicy.WAIT_UNTIL);
         IndexResponse indexResponse = client().index(indexRequest).actionGet();
         assertEquals(RestStatus.CREATED, indexResponse.status());
+        TaskId taskId = new TaskId("node", randomNonNegativeLong());
```

Comment:
`parentTaskId` ?

Reply:
Addressed in https://github.com/elastic/elasticsearch/pull/66206/commits/646aeab9ed596e00360112b72e628839aa98a47c